src/components/Types/Types.ts

```diff
@@ -0,0 +1,20 @@
+export interface Post {
+  body: string,
+  id?: number,
+  comments: Comment[],
+  title: string,
+  user: User,
+  userId?: number,
+}
+
+export interface Comment {
+  name: string,
+  email: string,
+  body: string,
+  id: number,
+}
+
+interface User {
+  name: string,
+  email: string,
+}
```

Comment:
Better to have separate files for types

Reply:
I think in this case it's the best option